modVM/build.xml

```diff
@@ -40,55 +41,12 @@
   </target>
 
   <target name="test_build" depends="build">
```

Comment:
why delete the contain of the ant task

Reply:
I put the lines back but CTRL+/ commented them out. modVM doesn't have tests yet and ant complains when it goes looking for them